canteen/templates/canteen/menu.html

```diff
@@ -57,7 +57,7 @@
         {% if menu.mutable %}
             <input class="checkout-button Grake-button" type="submit" value="提交">
             <div id="tips">
-                <p>本网站的服务由于<a href="http://guide.daocloud.io/dcs/%E4%BA%91%E7%AB%AF%E6%B5%8B%E8%AF%95%E7%8E%AF%E5%A2%83%E8%B5%84%E6%BA%90%E5%9B%9E%E6%94%B6%E8%AF%B4%E6%98%8E-9153734.html">主机商的限制</a>等原因，较为不稳定。如果遇到问题，烦请在浏览器访问 <a href="http://gzb.szsy.cn/card/">http://gzb.szsy.cn/card/</a> 学校官方订餐网站。</p>
+                <p>本网站的服务由于<a href="http://guide.daocloud.io/dcs/%E4%BA%91%E7%AB%AF%E6%B5%8B%E8%AF%95%E7%8E%AF%E5%A2%83%E8%B5%84%E6%BA%90%E5%9B%9E%E6%94%B6%E8%AF%B4%E6%98%8E-9153734.html">主机商的限制</a>等原因，较为不稳定。如果遇到问题，不妨尝试<a href="https://github.com/szes-maker/EAxT/releases/download/v0.2.1/org.szesmaker.ordermeal_v0.2.1.apk" target="_blank">食验客户端</a>，或烦请访问 <a href="http://gzb.szsy.cn/card/">http://gzb.szsy.cn/card/</a> 学校官方订餐网站。</p>
```

Comment:
Same as above. The detection script is more needed here.

Reply:
Or just change it to something like
```html
不妨尝试 <a href="">Android 客户端</a> 或烦请访问……
```
We have to avoid iOS users downloading an apk.

Reply:
I'm considering using [UAParser.js](https://github.com/faisalman/ua-parser-js) to identify user's OS, for it provides a handful of methods that might be useful besides this very occasion.
```
  var parser = new UAParser();
  switch(parser.getOS().name) {...}
```

Reply:
I made a design decision in the beginning that Javascript frameworks should be avoided to keep it simple and lightweight.
By now, only Google Analytics and [the calculator](https://github.com/szes-maker/canteen-ses/blob/fc922c2814c8b946dd2679101cc057136437948a/canteen/static/canteen/script/price_calc.js) will run on user's browser.
Although UAParser.js is small, we can just write a simple script which reads the user-agent string and test whether "iOS" is in it, in this use case.